requirements/test.txt

```diff
@@ -37,6 +37,8 @@ confluent-kafka[avro,schema-registry]==1.9.2
     # via -r requirements/test.in
 coverage[toml]==6.5.0
     # via pytest-cov
+ddt==1.6.0
+    # via -r requirements/test.in
```

Comment:
**Note to reviewer:** There is a separate issue around why `ddt` is getting added in the midst of the multiple "vias" that were under `coverage[toml]`.